src/components/JoinUs/OpenSource.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import remcalc from 'remcalc'
+import breakpoint from 'styled-components-breakpoint'
+import styled from 'styled-components'
+import getYoutubeId from 'get-youtube-id'
+import { Padding, Margin } from 'styled-components-spacing'
+import { CompensatedRow, CompensatedCol } from '../grid'
+import Image from '../Common/Image'
+import { Paragraph } from '../Typography'
+import playIcon from '../../images/button-play-default.svg'
+
+import { Section, Separator, TitleAndBody, TitleAndList } from './elements'
+
+const MAX_VIDEOS = 3
+
+const getThumbnail = videoId =>
+  `https://img.youtube.com/vi/${videoId}/maxresdefault.jpg`
+
+const TalkLink = styled.a`
+  border: 1px solid #eeefee;
+  display: block;
+  flex: 1;
+`
+const TalkTitle = styled.div`
+  display: flex;
+`
+const PlayIcon = styled.img`
+  min-height: ${remcalc(36)};
+  max-width: ${remcalc(36)};
+`
+const TruncatedParagraph = styled(Paragraph)`
+  max-height: 3rem;
+  overflow: hidden;
+
+  @supports (-webkit-line-clamp: 2) {
+    max-height: none;
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+`
+
+const TalkColumn = styled(CompensatedCol)`
+  ${breakpoint('largePhone')`
+    &:last-child {
+      display: none;
+    }
+  `}
+
+  ${breakpoint('tablet')`
+    &:last-child {
+      display: flex;
+    }
+  `}
+`
+
+const Talks = ({ data }) => (
+  <CompensatedRow>
+    {data.slice(0, MAX_VIDEOS).map(({ title, link }, idx) => (
+      <TalkColumn width={[1, 1, 1, 6 / 12, 6 / 12, 4 / 12]} key={idx}>
```

Comment:
Would be good to extract everything from this point downwards into a common component called StandAloneVideoLink, so that it can be used in other pages

Reply:
the `StandAloneVideoLink` is different — the design for these links have changed — and from what I understand, it has to be built on top of the `CompactVideoLink`. Currently, this link does not match with the design

Reply:
I don't really get it :disappointed: 

This is from the Join Us page on Zeplin:
![image](https://user-images.githubusercontent.com/15656538/51601289-dde66800-1efb-11e9-86a7-80a1fa31c7d3.png)

which is definitely making use of the StandAloneVideoLink component:
![image](https://user-images.githubusercontent.com/15656538/51601335-f6ef1900-1efb-11e9-8e7c-e17865b1d680.png)

Basing the StandAloneVideoLink on the CompactVideoLink makes sense :+1:

But either way, the component being used on this page is definitely the StandAloneVideoLink component, no? :confused:

Reply:
It is, but at the time of this discussion, the `StandAloneVideoLink` wasn't yet done. Now it's done and used appropriately